src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using System.Threading;
+using NuGet.Services.Configuration;
+
+namespace Ng.Jobs
+{
+    public class LoopingNgJob : NgJob
+    {
+        public LoopingNgJob(ILoggerFactory loggerFactory)
+            : base(loggerFactory)
+        {
+        }
+
+        public override async Task Run(IDictionary<string, string> arguments, CancellationToken cancellationToken)
+        {
+            var intervalSec = arguments.GetOrNull<int>(Arguments.Interval) ?? Arguments.DefaultInterval;
+
+            Logger.LogInformation("Looping job at interval {Interval} seconds", intervalSec);
+
+            Init(arguments, cancellationToken);
```

Comment:
Initializing very frequently will be a huge overhead for a job that runs continuosly